_pages/en/strategies-actions.md

```diff
@@ -40,49 +44,65 @@ Format:
+  - *Reorganize IT teams, starting with staff that are looking for a new challenge*
 - **IT teams and other groups must have access to modern tooling, such as SaaS (Internet), mobile and desktop tools**
   - *[Unblock access to specific SaaS for IITB employees](web-services-access.html)*
   - *Review the criteria and value add for technology bricks for SaaS*
   - *Invest in bandwidth, DLP and document classification tool to enable more open access to Internet*
-  - *Communications that employees can use SaaS on ESDC network for unclassified information only*
+  - *Communications that employees can use SaaS on ESDC network for unclassified information*
   - *Add all approved software to Application Catalogue (e.g. paying ones)*
   - *Automate parts of the software assessment and approval process*
   - *Share software certifications with other departments*
-- **IT teams must have access to resources enabling learning and automation of testing, deployments and other manual processes**
-  - *Temporary hiring of talent with programming skills (e.g. Co-op students, Casuals, [Talent Cloud](https://talent.canada.ca))*
-  - *Reorganize IT teams, starting with staff that are looking for a new challenge*
+
 - **[Hiring favours candidates who demonstrate adaptability, curiosity, and strong ethics](enable-learning.html)**
   - *Work with HR to adapt hiring processes (e.g. interview questions)*
-- **Create a safe environment for experimentation/failure and hold blameless postmortems**
-  - *Organize lunch between a team/project that recently failed and the DM*
-  - *Setup regular communications about failures and lessons learned*
-  - *Offer cloud sandboxes for experimentation*
+
+Notes:
+
 - Build multidisciplinary teams capable of delivering services from start to end of life
 - Encourages collaboration and fosters a culture of flexibility, mobility and well-being
 
 #### Process
 
 - **IT teams must use 10% of their time for improving existing processes and automation of testing, deployments and other manual processes**
-  - *Make it a requirement in performance agreements of managers and team leads, or mandatory for all IT teams*
+  - *Make it a requirement in performance agreements of managers and team leads*
+  - *Directors, management and teams must show quarterly progress related to automation and continuous improvements*
+  - *Track using CATS code (need code)*
   - *Communications about the updated rules around continuous improvements and automation*
+  - *Organize regular hackathons to improve and automate existing processes*
   - *Add automation tools to Application Catalogue*
   - *Create centralized repository of scripts on GCcode*
-  - *Assess RPA tools (Robotic Process Automation)*
   - *Add automation focus to [Grassroots Innovation Practice](http://dialogue/grp/IP/SitePages/Grassroots%20Innovation%20Practice.aspx) initiative*
-  - *Organize hackathons to automate existing processes*
+  - *Assess RPA tools (Robotic Process Automation)*
```

Comment:
Should we remove this?

Reply:
The RPA? I feel it's a bit prescriptive and related to a specific subset of ways to improve processes.

Reply:
Id remove it ... if only because I hate the term.

The association between robots and automation is damaging ... makes it seem far out, complicated, and something you need expertise to do. It's not. A simple bash script can save hundreds of hours. No robots or AI required.